main.go

```diff
@@ -37,6 +37,8 @@ var (
 
 	// profile the extension?
 	selfProfiling = getEnvBool("PYROSCOPE_SELF_PROFILING")
+
+	flushRelayQueue = getEnvBool("PYROSCOPE_FLUSH_RELAY_QUEUE")
```

Comment:
This is `false` by default, my understanding is that this functionality has really no clear downsides right? Shouldn't we make it `true` by default?

Reply:
The downside is that it can increase lambda invocation time and as a result billing. I'd keep it disabled by default. But it is not super strong opinion, we can change to enabled by default.

Reply:
Ahh I see now, since flush is sync, it makes the time before waiting for new events longer, therefore making it more expensive.

Reply:
maybe good idea to rename Flush to FlushSync so it is more obvious